src/query_param.c

```diff
@@ -195,3 +195,35 @@ int QueryParam_Resolve(Param *param, dict *params, QueryError *status) {
   }
   return -1;
 }
+
+int parseParams (dict **destParams, ArgsCursor *ac, QueryError *status) {
```

Comment:
Why this was moved here while `parseRequiredFields` was implemented elsewhere?
In general I am fine with moving code around but it should be done on a separate PR, here its hard to know if you made any changes to this function or just moved it...

Reply:
I moved it since the coordinator needs to use it. The coordinator now includes `query_param.h`

Reply:
And nothing was changed? Just moved?